todo_project/urls.py

```
@@ -1,12 +1,10 @@
 from django.urls import path, include
 from drf_spectacular.views import SpectacularAPIView, SpectacularSwaggerView, SpectacularRedocView
 
-urlpatterns = [
-    # Swagger UI at base path
-    path("", SpectacularSwaggerView.as_view(url_name="schema"), name="swagger-ui-base"),
+urlpatterns = [    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)